snapshot/snapshot.go

```diff
@@ -265,7 +268,11 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 		}
 	}
 
-	return o.mounts(ctx, s, parent)
+	start := time.Now()
+	dgst := digest.FromString("") // we pass the digest as an empty string here, since the metric we capture is layer agnostic
+	mounts, err := o.mounts(ctx, s, parent)
+	commonmetrics.MeasureLatency(commonmetrics.PrefetchesCompleted, dgst, start) // measuring the time the container launch is blocked on prefetch to complete
```

Comment:
Why not measure this in [`layer.WaitForPrefetchCompletion()`](https://github.com/containerd/stargz-snapshotter/blob/v0.7.0/fs/layer/layer.go#L90-L91)?

Reply:
> Why not measure this in [`layer.WaitForPrefetchCompletion()`](https://github.com/containerd/stargz-snapshotter/blob/v0.7.0/fs/layer/layer.go#L90-L91)?

We can't do that because it is only invoked if prefetch is enabled. Will you be OK with measuring it `Check` function defined in `fs.go`?

Reply:
> ill you be OK with measuring it in Check function defined in fs.go?

SGTM :)

Reply:
Updated.